### DISCUSSION TITLE
Can't get the still image for my Ring Doorbell

### DISCUSSION THREAD

**Discussion by twantythree** [tsightler/ring-mqtt#890] [Support Q&A]
Hi there, 

I've followed the steps to set up the camera integration using RTSP and the still image URL, but the still image is not working. Despite configuring everything according to the documentation, I’m unable to get the still image to display properly in the UI.

Based on the screenshots, it seems like I might have misconfigured something, but I’m not sure what. Any help would be greatly appreciated!
![1](https://github.com/user-attachments/assets/70b4a617-c75f-4880-a82d-189a1612f979)
![2](https://github.com/user-attachments/assets/cf72b2cd-e9a1-4bd3-a7b6-46f75a994c7d)

**Comment by tsightler** [comment]
My German is not very good, but I think that is the equivalent of "Error rendering template" in English and it tells you to check the logs for more info.  Did you do that?  What was that error (please copy and paste)?

Overall, the most likely issue is what is stated in the documentation, that the "guessed" entity ID listed in the Still Image URL attribute is not exactly correct.  You will need to find the actual entity ID for the snapshot camera in the HA UI and use that instead.  Easiest way to do that is to navigate to Developer Tools, click the States tab, and in the "filter entities" enter "camera.*_snapshot" which should list all of your snapshot camera entity IDs.  In the end, you are trying to get the template to resolve the entity_picture attribute for that device.

The addon attempts to guess this id based on the device name, but sometimes the HA algorithm does something unexpected.  Maybe umlauts are not allowed in entity IDs for example, or maybe HA is localizing entity IDs so "camera" is in German?  I'm not sure actually, but that's the most likely issue.  I wish there was an easier way but, since the addon runs in it's own container, it doesn't really have a way to find what entity ID HA actually assigns, at least I haven't been able to figure out a way to do it reliably.

**Comment by Dragon1473** [comment]
I have actually lost access to my still images as well.  They had been working fine since i set up the integration last year

**Reply by propi62** [reply]
keine umlaute. rename haustür to haustuer
